superset-frontend/packages/superset-ui-core/src/chart/index.ts

```diff
@@ -20,7 +20,11 @@
 export { default as ChartClient } from './clients/ChartClient';
 export { default as ChartMetadata } from './models/ChartMetadata';
 export { default as ChartPlugin } from './models/ChartPlugin';
-export { default as ChartProps, ChartPropsConfig } from './models/ChartProps';
+export { default as ChartProps } from './models/ChartProps';
+/* reexporting this interface as a type solves the dev-server
+ * warning the named export does not exist. See https://github.com/webpack/webpack/issues/7378#issuecomment-492641148
+ */
+export type { ChartPropsConfig } from './models/ChartProps';
```

Comment:
can you do something like this? It seems a bit more logical reading it this way, and may not need a comment then to explain why?
```
import { default as ChartProps, ChartPropsConfig } from './models/ChartProps';


export { ChartProps };
export type { ChartPropsConfig }; 
```

Reply:
certainly, the comment is there to justify the transfer from `interface` to `type` for future pedantic devs like me